src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/AbstractCloudFoundryDeployer.java

```diff
@@ -127,6 +130,21 @@ Path getApplication(AppDeploymentRequest request) {
 	}
 
 	/**
+	 * Return the Path to manifests.
+	 *
+	 */
+	Path getManifestsPath(AppDeploymentRequest request) {
+		try {
+			String manifestsPath = request.getDefinition().getProperties().get("manifests-path");
+			return (StringUtils.hasText(manifestsPath)) ? ResourceUtils.getFile(manifestsPath).toPath() : null;
```

Comment:
Also, I'd like to make sure the `getFile()` is the correct approach as this is in the context of where the SCDF server is running (on or off the CF environment)